R/data_interface.R

```diff
@@ -350,16 +350,14 @@ spark_data_write_generic <- function(df, path, fileMethod, mode = NULL, csvOptio
 #' @export
 spark_read_table <- function(sc,
                              name,
-                             path,
                              options = list(),
                              repartition = 0,
                              memory = TRUE,
                              overwrite = TRUE) {
 
   if (overwrite) spark_remove_table_if_exists(sc, name)
 
-  operation <- if (spark_version(sc) < "2.0.0") "load" else "table"
-  df <- spark_data_read_generic(sc, path, operation, options)
+  df <- spark_data_read_generic(sc, name, "table", options)
```

Comment:
@edgararuiz @kevinushey Alright, now we throw the following for Spark 1.6 and `master="local"`:

```
> spark_write_table(iris_tbl, name = "iris_named")
Error in spark_write_table.tbl_spark(iris_tbl, name = "iris_named") : 
  spark_write_table is not supported in local clusters for 1.6.0. Upgrade to Spark 2.X or use this function in a non-local Spark cluster.
```